### ISSUE TITLE
bug: inconsistent formatting due to short-circuiting black formatting

### ISSUE THREAD

**Issue by alkasm** [mkdocstrings/griffe2md#1]
### Description of the bug

There are inconsistencies in formatting due to short-circuiting running `black` formatting if used with the non-default config. In particular, a space is removed before equal signs in assignment expressions, and short functions don't have spaces around the -> between args and return type.

### To Reproduce

Given the following code to document:

```python
# griffe2mdtest.py
def short_function() -> int:
    pass


def function_with_many_args(
    a: int,
    b: str,
    c: dict[tuple[int, int], str],
) -> list[int]:
    pass


some_integer = 1
```

and the following script to generate markdown with a config updated to show type annotations:
```python
import griffe2mdtest

config = griffe2md.rendering.default_config
config["summary"] = False
config["show_signature_annotations"] = True
griffe2md.main.write_package_docs("griffe2mdtest", config=config)
```

The output is
````md
## griffe2mdtest

### griffe2mdtest.function_with_many_args

```python
function_with_many_args(
    a: int, b: str, c: dict[tuple[int, int], str]
) -> list[int]
```

### griffe2mdtest.short_function

```python
short_function()->int
```

### griffe2mdtest.some_integer

```python
some_integer= 1
```
````

Note the no spacing around `short_function()->int` and the spacing on only the right side for `some_integer= 1`.

### Expected behavior

I expect the following markdown to be generated:
````md
## griffe2mdtest

### griffe2mdtest.function_with_many_args

```python
function_with_many_args(
    a: int, b: str, c: dict[tuple[int, int], str]
) -> list[int]
```

### griffe2mdtest.short_function

```python
short_function() -> int
```

### griffe2mdtest.some_integer

```python
some_integer = 1
```
````

Removing the line length checks to short-circuit running black  in `griffe2md.rendering.do_format_code()` (for the attribute) and `griffe2md.rendering._format_signature()` (for the func sig) produces the expected output.

### Environment information

```bash
griffe2md --debug-info  # | xclip -selection clipboard
```

- __System__: macOS-14.4-arm64-arm-64bit
- __Python__: cpython 3.12.4 (/Users/alex/Library/Caches/pypoetry/virtualenvs/nominal-yyxi4Czq-py3.12/bin/python)
- __Environment variables__:
- __Installed packages__:
  - `griffe2md` v1.0.1

**Comment by pawamoy** [comment]
Hi @alkasm, thanks for the detailed report! I noticed this as well. Working on a fix right now.

**Comment by pawamoy** [comment]
Fix available in v1.0.2 :slightly_smiling_face:
